compiler/p/codegen/FPTreeEvaluator.cpp

```diff
@@ -453,7 +456,7 @@ TR::Register *OMR::Power::TreeEvaluator::vsplatsEvaluator(TR::Node *node, TR::Co
       return trgReg;
       }
 
-   TR_ASSERT(node->getDataType() == TR::VectorDouble, "unsupported splats type");
+   TR_ASSERT(node->getDataType().getVectorElementType() == TR::Double, "unsupported splats type");
```

Comment:
May as well make this a fatal assert.

Reply:
I would like to keep functional changes to the minimum in this PR. It was not FATAL assert before and I just mapped old interface into the equivalent new one.